parser/strconv.go

```diff
@@ -48,6 +50,19 @@ func ParseDuration(lit string) (*ast.DurationLiteral, error) {
 	}, nil
 }
 
+// FormatDuration will convert DurationLiteral into formatted string representation
+func FormatDuration(n *ast.DurationLiteral) string {
+	builder := strings.Builder{}
+	formatDuration := func(d ast.Duration) {
+		builder.WriteString(strconv.FormatInt(d.Magnitude, 10))
+		builder.WriteString(d.Unit)
```

Comment:
Why is this code abstracted through a function? Would it work to simply move the body of this function inside the loop?

Reply:
I copied the implementation from the existing ast formatter. 
I agree it doesn't make sense to abstract it through function. I will make a change now. 
Thanks for noticing.